src/EventListener/AddRateLimitHeadersListener.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bedrock\Bundle\RateLimitBundle\EventListener;
+
+use Bedrock\Bundle\RateLimitBundle\Model\StoredRateLimit;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+class AddRateLimitHeadersListener implements EventSubscriberInterface
+{
+    public function onKernelResponse(ResponseEvent $event): void
+    {
+        if ($event->getRequestType() !== HttpKernelInterface::MASTER_REQUEST) {
+            return;
+        }
+
+        $request = $event->getRequest();
+        if (!$request->attributes->has('_stored_rate_limit')) {
+            return;
+        }
+
+        $storedRateLimit = $request->attributes->get('_stored_rate_limit');
+
+        if (!$storedRateLimit instanceof StoredRateLimit) {
+            return;
+        }
+
+        $response = $event->getResponse();
+        $response->headers->set('x-rate-limit', (string) $storedRateLimit->getLimit());
+        $response->headers->set('x-rate-limit-hits', (string) $storedRateLimit->getHits());
+        $response->headers->set('x-rate-limit-until', $storedRateLimit->getValidUntil()->format('c'));
```

Comment:
Why we need to send this information ?

Reply:
I've just add a config to have a choice to display or not this information. 
This parameters is `display_headers` and the default value is `false`

Reply:
This information can be usefull to debug your configuration by route ;)